packages/msked/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "msked",
+  "version": "0.0.0",
+  "description": "Functions to operate with masked string values",
+  "keywords": ["masking", "input"],
+  "main": "src/index.ts",
```

Comment:
is it ok that it will have only Node build?

Reply:
Nope. You are right. I copied the wrong package. Thanks!